src/Engine/Manager/ScriptManager.cpp

```diff
@@ -782,18 +815,6 @@ int ScriptManager::GetStringDeclarationID() {
 
 }
 
-const int ScriptManager::GetSizeOfASType(int typeID, void* value) {
-
-   
-    if (typeID == asTYPEID_INT32)
-        return sizeof(int);
-    if (typeID == asTYPEID_FLOAT)
-        return sizeof(float);
-
-    return -1;
-
-}
-
 const std::vector<Component::Script*>& ScriptManager::GetScripts() const {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetScripts' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)